internal/server/network/driver_ovn.go

```diff
@@ -3399,6 +3399,11 @@ func (n *ovn) Rename(newName string) error {
 // chassisEnabled checks the cluster config to see if this particular
 // member should act as an OVN chassis.
 func (n *ovn) chassisEnabled(ctx context.Context, tx *db.ClusterTx) (bool, error) {
+	// If parent is "none", this network is standalone and should not act as a chassis.
+	if n.config["parent"] == "none" {
+		return false, nil
+	}
```

Comment:
This check is incorrect. `n.config` here refers to the config of the OVN network when the `parent` check needs to be done on the uplink network instead.

So you should change the check to:
 - Check that we have an uplink network at all `d.config["network"] != "none"`
 - Check that the uplink network is of type `physical`
 - Check if the `parent` property on the uplink network is set to `none`